src/main/msp/msp.c

```diff
@@ -2276,6 +2336,14 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             writeEEPROM();
         }
         readEEPROM();
+
+#ifdef USE_VTX_TABLE
+        if (vtxTableNeedsInit) {
+            vtxTableNeedsInit = false;
+            vtxTableInit();  // Reinitialize and refresh the in-memory copies
```

Comment:
Looking at this, should we make the `vtxTableInit()` part of what is done as part of the `readEEPROM()` in any case?

Reply:
I thought about that and specifically looked at putting it in `activateConfig()`. But looking at `init()` the `readEEPROM()` is done quite early in the process while the existing call to `vtxTableInit()` is quite late so I wasn't confident that enough other initialization was completed at the point `readEEPROM()` was called during startup.

Reply:
Yeah, `init()` is a bit of a chicken and egg situation with respect to `readEEPROM()`, as we need it to be early to know what we need to initialise.